### DISCUSSION TITLE
Question about analytics engine customization

### DISCUSSION THREAD

**Discussion by dheerajsharma14** [CloudMonitorFinOps/reporting#15] [General]
Hi, I see that the various PBIX reports are available in the Open Source category. However to setup the overall Cloud Monitor tool, it seems like this PBIXs reports runs over the output produced by Analytics Engine. Whatever I have understood so far.

So, if I have to take a look at that source code, is it open source and available in a different repo? If not, what can I leverage with these PBIX without the Analytics Engine?

Regards,
Dheeraj

**Comment by rodneyjoyce** [comment]
Hi Dheeraj,

That is correct - you need to connect to the CloudMonitor Analytics Engine. The Engine is not open-source as this is our IP.
